engine/physics/src/physics/test/test_physics_2d.cpp

```diff
@@ -175,6 +177,28 @@ TYPED_TEST(PhysicsTest, MultipleGroups)
     (*TestFixture::m_Test.m_DeleteCollisionShapeFunc)(shapes[1]);
 }
 
+TYPED_TEST(PhysicsTest, UseBullet)
+{
+    VisualObject vo_b;
+    vo_b.m_Position = Point3(0.5f, 3.0f, 0.0f);
+
+    dmPhysics::CollisionObjectData data;
+    data.m_Type = dmPhysics::COLLISION_OBJECT_TYPE_DYNAMIC;
+    data.m_Mass = 1.0f;
+    data.m_UserData = &vo_b;
+    data.m_Group = 1 << 3;
+    data.m_Mask = 1 << 2;
+    typename TypeParam::CollisionShapeType shape = (*TestFixture::m_Test.m_NewBoxShapeFunc)(TestFixture::m_Context, Vector3(0.5f, 0.5f, 0.0f));
+    typename TypeParam::CollisionObjectType dynamic_co = (*TestFixture::m_Test.m_NewCollisionObjectFunc)(TestFixture::m_World, data, &shape, 1u);
+
+    (*TestFixture::m_Test.m_IsBulletFunc)(dynamic_co);
```

Comment:
Is there a reason you don't check the return values of the function?

Reply:
I thought the test was mostly about calling the functions and assuring they don't produce a link error. Let me fix...